mrp_partial_production/models/mrp.py

```diff
@@ -40,13 +44,16 @@ def get_qty_available_to_produce(self):
         lines = self.bom_id.explode(self.product_id, quantity)[1]
 
         result, lines_dict = defaultdict(int), defaultdict(int)
-        for res in self.move_raw_ids:
-            result[res.product_id.id] += res.reserved_availability
+        for res in self.move_raw_ids.filtered(
+                lambda move: move.is_done is False):
+            result[res.product_id.id] += (res.reserved_availability -
+                                          res.quantity_done)
 
         for line, line_data in lines:
             lines_dict[line.product_id.id] += line_data['qty'] / quantity
 
-        qty = [(int(result[key] / val)) for key, val in lines_dict.items()
+        qty = [(result[key] / val) for key, val in lines_dict.items()
```

Comment:
Why did you change this?

Reply:
For reasons of rounding there were test cases where the calculation failed when different products, when they had very close quantities, the ```min``` function did not take the correct value since they all had the same value